requirements.txt

```diff
@@ -1,5 +0,0 @@
-fastapi[standard]>=0.113.0,<0.114.0
-psycopg2
-pydantic>=2.7.0,<3.0.0
-redis
-sqlmodel
```

Comment:
Do we actually use sqlmodel?

Reply:
We use sqlalchemy which is a part of sqlmodel. We could reduce this dependency to sqlalchemy instead, good catch.